apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -3957,8 +3440,8 @@ describe('[Users]', function () {
 	});
 
 	describe('[/users.autocomplete]', () => {
-		after(() => {
-			updatePermission('view-outside-room', ['admin', 'owner', 'moderator', 'user']);
+		after(async () => {
```

Comment:
no need to do this, just remove the `{` and let the arrow function return the promise
```javascript
after(() => updatePermission('view-outside-room', ['admin', 'owner', 'moderator', 'user']));
```